rusty-machine/tests/learning/k_means.rs

```diff
@@ -61,13 +61,13 @@ fn test_model_ran_partition() {
 #[test]
 fn test_model_kplusplus() {
     let mut model = KMeansClassifier::new(3);
-    let data = Matrix::new(3, 2, vec![1.0, 2.0, 1.0, 3.0, 1.0, 4.0]);
+    let inputs = Matrix::new(3, 2, vec![1.0, 2.0, 1.0, 3.0, 1.0, 4.0]);
     let pred_data = Matrix::new(3,2, vec![1.0, 2.0, 1.0, 3.0, 1.0, 4.0]);
```

Comment:
Strictly speaking we should update `pred_data` to `targets` too. There's quite a few different terms present in the tests - that's my bad!